### ISSUE TITLE
Compass, logo, info button have ambiguous vertical positions

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-gl-native#5549]
In iosapp, the compass, logo view, and ℹ️ button each have ambiguous vertical positions at runtime, as reported by Xcode 8:

```
Layout Issues Group
x-xcode-debug-views://7fe5680d30f0: runtime: Layout Issues: Vertical position is ambiguous for UIImageView.

x-xcode-debug-views://7fe5680d30f0: runtime: Layout Issues: Vertical position is ambiguous for UIButton.

x-xcode-debug-views://7fe5680d30f0: runtime: Layout Issues: Vertical position is ambiguous for UIView.
```

/cc @friedbunny

**Comment by 1ec5** [comment]
Copying milestone from #6260. More details there.

**Comment by frederoni** [comment]
Fixed in #6776
